projboard/tests/test_user.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from projboard.models import User
+
+
+NAME = "RAWAD"
+NICKNAME = "R1998"
+EMAIL = "rawad@gmail.com"
+PASSWORD = "rawad!@#"
+
+
+@pytest.fixture
+def generate_user():
+    user1 = User.create_user(name="RAWAD", nickname="R1998", email="rawad@gmail.com", password="rawad!@#")
```

Comment:
wouldn't user1 be more readable as `User.create_user(name=NAME, nickname=NICKNAME, email=EMAIL, password=PASSWORD)` ?

Reply:
Done